usecases/sscx/pathways/projection/TC-L6_FS.yaml

```diff
@@ -1,19 +1,19 @@
 reference:
-    author: "Bruno and Sakmann 2006"
+    author: "Beierlein and Connors 2002 (extracellular minimal stimulation)"
     psp_amplitude:
-        mean: 0.95
-        std: 1.1
+        mean: 3.9
+        std: 3.5
 
 pathway:
-    projection: Thalamocortical_input_VPM
+    projection: VPM
```

Comment:
Specifying `projection` is not mandatory (nor should it have any effect) since the edge population is given as a command line parameter.

It doesn't hurt to have it here, though. Just a reminder.

Reply:
good point!